docs/usage.rst

```diff
@@ -118,9 +118,12 @@ Making jobs' output available to ``docker logs`` of the executing container
 
 Docker captures the output of the first process in a container as logged data. In order to capture
 the output of a job's command as well, its output needs to be redirected to the main process'
-``stdout`` or ``stderr``, e.g. with by redirecting a command's output with a shell::
+``stdout`` and ``stderr``, e.g. by redirecting a command's output with a shell::
 
-    deck-chores.a_job.command: sh -c "/usr/local/bin/job_script.sh &> /proc/1/fd/1"
+    deck-chores.a_job.command: sh -c "/usr/local/bin/job_script.sh > /proc/1/fd/1 2> /proc/1/fd/2"
+
+It is important to redirect both ``stdout`` (``>``) AND ``stderr`` (``2>``) separately so 
```

Comment:
the importance depends on what one wants to achieve, please reformulate to make that clear. there's no reason to put 'and' into allcaps.

Reply:
Fair point, I'll update it

Reply:
Done, new commit added.